cli/src/main/kotlin/OrtMain.kt

```diff
@@ -71,7 +71,10 @@ class OrtMain : CliktCommand(name = ORT_NAME, epilog = "* denotes required optio
 
     private val stacktrace by option(help = "Print out the stacktrace for all exceptions.").flag()
 
-    private val configArguments by option("-P", help = "Allows to override configuration parameters.").pair().multiple()
+    private val configArguments by option("-P", help = "Allows to override configuration parameters.")
+        .convert { Pair(it.substringBefore("="), it.substringAfter("=", "")) }
```

Comment:
How about using

    .convert { keyValue -> keyValue.split("=").let { it[0] to it.getOrElse(1) { "" } } }

which only searches for `=` in the option.argument once?

Reply:
The implementation `as-is` can handle '=' as value while your proposal can't. Performance considerations imo are not relevant. What do you think?

Reply:
Ok.